### PR TITLE
Added docs about api_version

### PR DESCRIPTION
**Pull request by Nyholm**
Added docs as requested in #415

### CODE REVIEW

**Comment by cursedcoder** [comment]
Would be good to cover a ctor method from https://github.com/KnpLabs/php-github-api/pull/419/files

File: doc/api_version.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Api version
+[Back to the navigation](README.md)
+
+If you want to change the API version from its default ("v3") you may do that with a constructor argument.
+For example:
+
+```php
+$client = new Github\Client();
+echo $client->getApiVersion(); // prints "s3"
```
Comment (GrahamCampbell) [inline doc/api_version.md]: s3?

Comment (Nyholm) [inline doc/api_version.md]: haha, Im also working with AWS right now..

**Comment by Nyholm** [comment]
These docs assumes that PR #423 will be merged.

**Comment by Nyholm** [comment]
Thank you for merging

**Pull request merged**